### ISSUE TITLE
Export current scene feature

### ISSUE THREAD

**Issue by HeadClot** [rdeioris/glTFRuntime#1]
Would it be possible to get an export current scene option for GLTFRuntime? 

My use case is that I am making scenes for Social VR platforms that accept .GLTF and .GLB.

**Comment by rdeioris** [comment]
@HeadClot yes it is possible (we already have the exporting infrastructure for skeletons). It would be a pretty big work, so eventually ping Epic about our request for glTFRuntime megagrant, or if you can sponsor the effort just contact me on the discord channel.
